test/session.test.js

```diff
@@ -248,7 +247,6 @@ test('should decryptSession with custom request object', async (t) => {
   const requestObj = {}
   fastify.decryptSession(sessionId, requestObj, () => {
     t.equal(requestObj.session.cookie.maxAge, null)
-    t.equal(requestObj.session.sessionId, DEFAULT_SESSION_ID)
```

Comment:
Hmm. Does the sessionid now exists or not? Instead of removing the assertions i prefer to keep them if they are not correct anymore i prefer that they are changed.

Reply:
Yeah I did this a while ago because it was unrelated to the test. I'll re-add